frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -333,6 +335,15 @@ export const funnelLogic = kea<funnelLogicType>({
                 excludeProperty: (state, { propertyName }) => [...state, propertyName],
             },
         ],
+        excludedEventNames: [
+            [] as string[],
+            {
+                persist: true,
+            },
+            {
+                excludeEvent: (state, { eventName }) => [...state, eventName],
```

Comment:
non-blocking: Can we call it something more descriptive than state? That may be how its modelled in the abstract sense but we have a concrete use case.